core/commands/storage.go

```diff
@@ -1158,7 +1157,12 @@ var storageAnnounceCmd = &cmds.Command{
 	Helptext: cmds.HelpText{
 		Tagline: "Update and announce storage host information.",
 		ShortDescription: `
-This command updates host information and broadcasts to the BTFS network.`,
+This command updates host information and broadcasts to the BTFS network. 
+
+Examples
+
+To set the max price per GiB to 1 BTT:
+$ btfs storage announce --host-storage-price 1`,		
```

Comment:
I think you need `--host-storage-price=1`

Reply:
@taiyangc This is updated now with `--host-storage-price=1`